src/app/state/enrolments/owned/owned.selectors.spec.ts

```diff
@@ -0,0 +1,72 @@
+// import * as OwnedSelectors from './owned.selectors';
+//
+// describe('Owned Assets Selectors', () => {
+//   const asset: any = {
```

Comment:
Why is it commented?

Reply:
Added few tests to owned enrolments and requested enrolments selectors.